docs/source/Howtos/Beginner-Tutorial/Part3/Beginner-Tutorial-Rules.md

```diff
@@ -584,7 +584,7 @@ class TestEvAdventureRuleEngine(BaseEvenniaTest):
     @patch("evadventure.rules.randint")
     def test_roll(self, mock_randint):
         mock_randint.return_value = 4 
-        self.assertEqual(self.roll_engine.roll("1d6", 4))     
+        self.assertEqual(self.roll_engine.roll("1d6"), 4)     
```

Comment:
`.roll` is only expecting the "1d6" argument, 4 is supposed to be the second argument for `.assertEqual`. Gave a TypeError when run as-is.